competencies/roles/Developer.md

```diff
@@ -63,6 +63,7 @@ As well as the role specific competencies below, all tech roles in properly invo
             <li>Actively develops and tests for relevant system issues (what happens in failure, how is the latency)</li>
             <li>Routinely considers most important system characteristics in designs (e.g. latency vs throughput vs resilience)</li>
             <li>Routinely raises system relevant system design factors, and helps team disregard irrelevant (latency on background process, testability of one time script)</li>
+            <li>Manage the plan, start, execution, end, and maintenance of at least one project of a significant system</li>
```

Comment:
I know it's just a single phrase, but I have feelings about different parts of it. 😄 

- "Manage the plan, ..." sounds weird and vague to me (are you really doing the planning? or are you managing someone who's planning? and after the planning is done, what's your level of involvement?). Maybe just a language thing, but I guess my expectation around this was more around "Plan, oversee and own ..."
- I think "deployment" is a word to be added to the list of things there, as deployment of new complex systems often require orchestration of different pieces that speak to this proficiency level
- I tend to prefer more tangible/measurable goals, so maybe "significant system" could be replaced by something like "a system involving more than X services"
- I think a "or an overhaul of a similar system" would also fit this category (for instance, rethinking/reimplementing an existing system would exercise the same skills)